jwst/cube_build/ifu_cube.py

```diff
@@ -395,7 +395,7 @@ def set_geometryAB(self, corner_a, corner_b, lambda_min, lambda_max):
             across_cdelt = self.cdelt2
 
         if self.instrument == 'NIRSPEC':
-            self.cdelt2 = self.cdelt1  # make cubes square
+            # self.cdelt2 = self.cdelt1  # make cubes square
```

Comment:
Probably should just remove this line, instead of leaving it comment out.

Reply:
Done.